include/net/net_pkt.h

```diff
@@ -687,7 +687,7 @@ static inline void net_pkt_set_src_ipv6_addr(struct net_pkt *pkt)
 	NET_BUF_POOL_DEFINE(name, count, CONFIG_NET_BUF_DATA_SIZE,	\
 			    CONFIG_NET_BUF_USER_DATA_SIZE, NULL)
 
-#if defined(CONFIG_NET_DEBUG_NET_PKT)
+#if CONFIG_NET_LOG_LEVEL_NET_PKT > 3
```

Comment:
I think we should use LOG_* constants instead of numeric values.

Reply:
And probably use >= comparison for clarity. ("Logging starts at the level X", not "logging starts after the level Y".)